staging/src/k8s.io/kubectl/pkg/cmd/top/top.go

```diff
@@ -39,9 +39,27 @@ var (
 	topLong = templates.LongDesc(i18n.T(`
 		Display resource (CPU/memory) usage.
 
-		The top command allows you to see the resource consumption for nodes or pods.
+		This command provides a real-time view of resource consumption for nodes and pods.
+		It fetches metrics from the Metrics Server, which aggregates this data from the
+		kubelet on each node. The Metrics Server must be installed and running in the
+		cluster for this command to work.
 
-		This command requires Metrics Server to be correctly configured and working on the server. `))
+		This command is useful for observing current resource loads. Because it displays
+		the same metrics used by the Horizontal Pod Autoscaler (HPA) to make scaling
+		decisions, you can use it to directly validate and debug your HPA configurations.
```

Comment:
Would mention that metrics collected optimized for HPA/VPA decision, and not accuracy, so the values might not match with standard OS tools.

I'm alluding to memory returned being based on "working set" which is different than standard tools return https://github.com/kubernetes-sigs/metrics-server/blob/master/FAQ.md#how-memory-usage-is-calculated